pkg/kubefed2/enable/enable.go

```diff
@@ -346,25 +345,8 @@ func writeObjectToYAML(obj pkgruntime.Object, w io.Writer) error {
 		return err
 	}
 
-	// Convert to unstructured to filter status out of the output.  If
-	// status is included in the yaml, attempting to create it in a
-	// kube API will cause an error.
+	// Use unstructured to filter unwanted fields out of the output.
```

Comment:
nit: this comment doesn't seem relevant given that removal of unwanted fields is delegated to yaml conversion.

Reply:
Updated!